app/javascript/src/components/menus/activated_menu_activator.js

```diff
@@ -35,9 +35,13 @@ class ActivatedMenuActivator {
     this.#config = config;
     this.#className = "ActivatedMenuActivator";
     this.menu = menu;
+    
```

Comment:
The three dots are a stylistic addition and part of the design, so should be controlled by the CSS and not become part of the code in the component.